content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Implementation status"
+linkTitle: "Implementation status"
+weight: 8
+---
```

Comment:
Maybe add a legend:
✅ supported
❌ not supported 
[blank] no data 

The main goal being to clarify the difference between missing information and not supported feature.

Reply:
Proposed addition (targeting this PR) in https://github.com/alippai/parquet-site/pull/1